src/MICore/Debugger.cs

```diff
@@ -505,6 +534,12 @@ private bool IsLocalGdb()
             }
         }
 
+        private bool IsRemoteGdb()
+        {
+            return this.MICommandFactory.Mode == MIMode.Gdb &&
+               this._launchOptions is PipeLaunchOptions;
```

Comment:
Although I am guessing in your case you really do only want to handle pipe, so maybe this should method should just be renamed.

---
In reply to: [64105587](https://github.com/Microsoft/MIEngine/pull/329#discussion_r64105587) [](ancestors = 64105587)